packages/atomic/src/components/search/facets/atomic-facet/e2e/atomic-facet.e2e.ts

```diff
@@ -1,3 +1,5 @@
+import {FacetSortCriterion} from '@coveo/headless';
+import {orderBy} from 'natural-orderby';
```

Comment:
I don't think adding a dependency is necessary for tests. Is there other ways you can test this ? Or at least put it in devDependencies ?

Reply:
Yep def should be in dev deps. Extracting the sorting logic from natural-orderby would be too much work. 

I tried with natural-compare-lite and javascript-natural-sort, which are already in our deps, but they don't sort in the same way as the index...

Reply:
How do we test the other sorts?

Reply:
@louis-bompart for score, not sure how we could test since the score of facet values is not included in the response. 

I did however add some tests for custom-sort :)

Reply:
Added some for 'automatic' too